.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```

Comment:
The dependabot docs don't give much insight, but I am going to assume this does not attempt to automatically propose new Go versions, only the dependency updates?

Reply:
As far as I know only dependencies and not a major golang upgrade... but even if such a PR would be created you wouldn't have to merge it 😉

Reply:
For sure and agreed 😄